src/frontend/next/src/components/Post/Post.tsx

```diff
@@ -0,0 +1,210 @@
+import React, { useRef, useState } from 'react';
+
+import useSWR from 'swr';
+import 'highlight.js/styles/github.css';
+import { makeStyles } from '@material-ui/core/styles';
+import { Box, Grid, Typography, ListSubheader, createStyles } from '@material-ui/core';
+
+// For style, need to add:
+// import '../styles/telescope-post-content.css';
+
+interface Props {
+  postUrl: string;
+}
+
+interface DataItem {
```

Comment:
This is really a `Post` vs. `DataItem`.  We probably should create something like this: https://github.com/vercel/next.js/tree/canary/examples/with-typescript/interfaces and move `interface Feed {...}` and `interface Post {...}` in there, then import them into this file so we can use them.